packages/ui/src/molecules/Incentives/IncentivesPure.tsx

```diff
@@ -0,0 +1,30 @@
+import type { HTMLAttributes } from 'react'
+import React, { forwardRef } from 'react'
+
+export interface IncentivesPureProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+}
+
+const IncentivesPure = forwardRef<HTMLDivElement, IncentivesPureProps>(
```

Comment:
I didn't get why we need this component. This isn't exported either as far as I can see